### PR TITLE
Fix margin calculation

### PR DESCRIPTION
**Pull request by KaSroka**
Reopen as #145 seems to be broken (new commits are not appearing)

### CODE REVIEW

**Comment by KaSroka** [comment]
Change in nrf_raal_softdevice.c:180

File: src/nrf_802154_utils.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+/**
+ * @defgroup nrf_802154_utils Utils definitions used in the 802.15.4 driver.
+ * @{
+ * @ingroup nrf_802154
+ * @brief Definitions of utils used in the 802.15.4 driver.
+ */
+
+/**@brief RTC clock frequency. */
+#define NRF_802154_RTC_FREQUENCY             32768UL
```
Comment (LuDuda) [inline src/nrf_802154_utils.h]: nit: alignment seems to be strange

**Comment by hubertmis** [comment]
Regression test

**Comment by LuDuda** [comment]
Nice catch :+1:

**Pull request merged**